recipes/gperftools/all/test_package/conanfile.py

```diff
@@ -19,7 +21,20 @@ def build(self):
         cmake.configure()
         cmake.build()
 
+    def _test(self, executable):
+        bin_path = os.path.join(self.cpp.build.bindir, executable)
+        if conan_version.major >= "2.0.15":
+            stderr = io.StringIO()
+            kwargs = {"stderr".strip(): stderr}
+            self.run(bin_path, env="conanrun",  **kwargs)
```

Comment:
```suggestion
            self.run(bin_path, env="conanrun",  stderr=stderr)
```

Reply:
Would love to do it that way, but even with the current obfuscation, the linter still complains about `stderr` being an unsupported keyword argument. :/

Reply:
It's just an opinion, so do as other reviewers will say.
As far as I'm concerned, If there's a linter message anyway I prefer the one I can understand. At least the code is more readable with the suggestion.